tools/cmake/requirements.txt

```diff
@@ -1,3 +1,4 @@
 prettytable==0.7.2
 future==0.16.0
 Jinja2>=2.10.1,<2.11
+intelhex
```

Comment:
@multiplemonomials could you specify the version details for _intelhex_ module as like other dependencies here

Reply:
Done!